parley/src/layout/editor.rs

```diff
@@ -442,6 +464,18 @@ where
         self.buffer.clone().into()
     }
 
+    /// Get the current generation of the layout, to decide whether to draw
+    pub fn generation(&self) -> usize {
+        self.generation
+    }
+
+    /// Update the layout if it is dirty
```

Comment:
Clean feels like an odd word to me here

Reply:
I could use the word `refresh` like we do on `Selection`, if that is preferred.